inference-engine/tests/functional/plugin/cpu/single_layer_tests/ctc_greedy_decoder.cpp

```diff
@@ -0,0 +1,151 @@
+        ElementType inType;
+        bool mergeRepeated;
+        InputShapeParams shapes;
+        std::tie(shapes, inType, mergeRepeated) = GetParam();
+
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        inputDynamicShapes = shapes.first;
+
+        for (auto& shape : shapes.second) {
+            size_t T;
+            size_t N;
+            size_t C;
+            std::tie(T, N, C) = shape;
+            targetStaticShapes.push_back({{T, N, C}, {T, N}});
+        }
+
+        auto params = ngraph::builder::makeDynamicParams(inType, inputDynamicShapes);
+        auto ctcGreedyDecoder = std::make_shared<ov::op::v0::CTCGreedyDecoder>(params[0], params[1], mergeRepeated);
+
+        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(ctcGreedyDecoder)};
+        function = std::make_shared<ngraph::Function>(results, params, "CTCGreedyDecoderCPU");
+    };
+
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (int i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            ov::runtime::Tensor tensor;
+            if (i == 0) {
+                if (funcInput.get_element_type().is_real()) {
+                    tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(),
+                                                                     targetInputStaticShapes[i],
+                                                                     10,
+                                                                     0,
+                                                                     1000);
+                } else {
+                    tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(),
+                                                                     targetInputStaticShapes[i]);
+                }
+            } else {
+                auto T = targetInputStaticShapes[i][0];
+                auto B = targetInputStaticShapes[i][1];
+                std::mt19937 gen(1);
+                std::uniform_int_distribution<unsigned long> dist(1, T);
+
+                std::vector<float> sequenceMaskData(B * T, 0);
+                for (int b = 0; b < B; b++) {
+                    int len = dist(gen);
+                    for (int t = 0; t < len; t++) {
+                        sequenceMaskData[t * B + b] = 1;
+                    }
+                }
+                tensor = ov::runtime::Tensor{funcInput.get_element_type(), targetInputStaticShapes[i]};
+                float* begin = tensor.data<float>();
+                std::copy(sequenceMaskData.begin(), sequenceMaskData.end(), begin);
+            }
+
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
+    }
+};
+
+TEST_P(CTCGreedyDecoderLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    run();
```

Comment:
also add CheckPluginRelatedResults(executableNetwork, "CTCGreedyDecoder");

Reply:
CheckPluginRelatedResults() method currently doesn't support `ov::runtime::ExecutableNetwork` API yet. will add a *TODO* here for reminder

Reply:
after merge this PR it supported https://github.com/openvinotoolkit/openvino/pull/8339
uncomment please